test/lit.cfg

```diff
@@ -833,3 +853,6 @@ config.substitutions.append(('%target-object-format', config.target_object_forma
 
 config.substitutions.append(('%target-resilience-test-wmo', config.target_resilience_test_wmo))
 config.substitutions.append(('%target-resilience-test', config.target_resilience_test))
+
+config.substitutions.append(('%target-llvm-profdata', os.path.join(llvm_tools_dir, 'llvm-profdata')))
+config.substitutions.append(('%target-llvm-cov', os.path.join(llvm_tools_dir, 'llvm-cov')))
```

Comment:
Is anyone using these, though? Do you expect anyone to set them from the command line? If not, you should just ignore them altogether, like FileCheck.